whirlpool/appliance.py

```diff
@@ -122,6 +128,21 @@ def bool_to_attr_value(self, b: bool):
     def attr_value_to_bool(self, val: str):
         return None if val is None else val == SETVAL_VALUE_ON
 
+    def attr_value_to_int(self, val: str):
+        value = 0
+        try:
+            value = int(val)
+        except ValueError:
+            return 0
+        return value
+
+    def attr_value_secs_to_mins_int(self, val: str):
+        val_int = self.attr_value_to_int(val)
+        mins = math.floor(val_int/60)
+        
+        return mins
+
+
```

Comment:
```suggestion
```
The lib should just return secs. HA will take care of converting to minutes if it makes sense there.